spec/ysh-word-eval.test.sh

```diff
@@ -92,18 +92,9 @@ true
 3.14
 ## END
 
-#### @[range()]
-shopt -s oil:all
-write @[range(10, 15, 2)]
-## STDOUT:
-10
-12
-14
-## END
```

Comment:
Removed this since there is no more builtin to implicitly convert a range into a list. There was some code in `Stringify()` that explicitly rejects iterator-like things (like dict keys), so I figure ranges should be treated the same way